src/main/java/seedu/tinner/logic/commands/AddRoleCommand.java

```diff
@@ -27,11 +27,11 @@
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a role to the company listed. "
             + "Parameters: "
             + "companyIndex "
-            + PREFIX_NAME + "ROLE_NAME (TYPE) "
+            + PREFIX_NAME + "ROLE_NAME [(TYPE)] "
             + PREFIX_STATUS + "STATUS "
-            + PREFIX_REMINDER_DATE + "REMINDER_DATE "
-            + PREFIX_DESCRIPTION + "DESCRIPTION "
-            + PREFIX_STIPEND + "STIPEND\n"
+            + PREFIX_REMINDER_DATE + "[REMINDER_DATE] "
```

Comment:
I feel that the brackets here should be across both the prefix and the value to be more specific

Reply:
Oops! thanks for spotting. Made the changes.